sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -215,7 +212,7 @@ function processFormData(formData?: FormDataMap) {
   for (const element in formData) {
     const item = formData[element];
     if (item instanceof Uint8Array) {
-      processedFormData[element] = binaryArrayToString(item);
+      processedFormData[element] = new Blob([item]);
```

Comment:
do we know if anyone actually needs this? shouldn't we require client implementations to call `createFile` directly in cases a blob needs to be created?

Reply:
This is a valid question. Purview does expose this here: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/purview/purview-catalog-rest/review/purview-catalog.api.md?plain=1#L1897, so requiring File would be breaking. I am inclined to preserve the current behavior for now to unblock people and then revisit as part of the wider investigation into how we should be handling binary data in general.

Reply:
ah I see. We may want to add an inline comment to that effect here as a service to our future selves 😅

Reply:
yeah, I can add a comment. I think there's definitely room for improvement here. It ties in to the general lack of streaming support for upload (ironically, Purview's property is called `uploadedInputStream` while not supporting a stream...).